pkg/operator/operator_config.go

```diff
@@ -303,9 +303,12 @@ func (r *operatorConfigReconciler) makeRuleEvaluatorDeployment(spec *monitoringv
 	if r.opts.CloudMonitoringEndpoint != "" {
 		evaluatorArgs = append(evaluatorArgs, fmt.Sprintf("--export.endpoint=%s", r.opts.CloudMonitoringEndpoint))
 	}
-	if spec.QueryProjectID != "" {
-		evaluatorArgs = append(evaluatorArgs, fmt.Sprintf("--query.project-id=%s", spec.QueryProjectID))
+	queryProjectID := r.opts.ProjectID
```

Comment:
Can you add a comment explaining why we want to default to the operator's project?